packages/combobox/src/ComboboxOption/ComboboxOption.styles.ts

```diff
@@ -46,12 +42,14 @@ export const checkIconStyle: Record<ComboboxSize, string> = {
   `,
 };
 
-export const flexSpan = css`
-  display: inline-flex;
-  gap: 8px;
-  justify-content: start;
-  align-items: inherit;
-  overflow-wrap: anywhere;
+export const checkBoxBaseStyles = css`
+  pointer-events: none;
+  gap: 0;
+
+  label {
```

Comment:
can we use createUniqueClassName on the label and then use a higher specificity here to target the element that we want to style?

Reply:
This label is from the `CheckBox` component. I don't know if it makes sense to add a unique classname to that. If a consumer wanted to tweak some CSS inside the `CheckBox` component, they would go with the className prop approach.